File: src/main/java/uk/gov/justice/hmpps/prison/api/model/MilitaryRecords.java
package uk.gov.justice.hmpps.prison.api.model;

import io.swagger.v3.oas.annotations.media.Schema;
import io.swagger.v3.oas.annotations.media.Schema;
import io.swagger.v3.oas.annotations.Hidden;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Schema(description = "Military Records")
@Data
@AllArgsConstructor
public class MilitaryRecords {
    @Schema(description = "Military Records")
    final List<MilitaryRecord> militaryRecords;
}

File: src/main/java/uk/gov/justice/hmpps/prison/api/model/MilitaryRecords.kt
package uk.gov.justice.hmpps.prison.api.model

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "Prisoner military records with details about service in the UK Armed Forces.")
data class MilitaryRecords(
  @Schema(description = "List of military records.")
  val militaryRecords: List<MilitaryRecord>,
)
